packages/cli/yaml/docs-validator/src/ValidationViolation.ts

```diff
@@ -2,6 +2,7 @@ import { NodePath } from "@fern-api/fern-definition-schema";
 import { RelativeFilePath } from "@fern-api/fs-utils";
 
 export interface ValidationViolation {
+    name?: string;
```

Comment:
Fine as-is for now - in the future I bet we'll make this required.

Reply:
Yeah I was thinking that'd be good to do either as a follow-up or just incrementally so that this PR doesn't get even more huge